db/seeds.rb

```diff
@@ -124,13 +124,13 @@
                         brand: 'K2',
                         image: File.open('app/assets/images/fastplant.png')
                       },
-                      { name: 'Burton Process Snowboard',
+                      { name: 'Burton Process',
                         description: 'Take your freestyle further with the snowboard Mark McMorris chooses for lightweight performance and action-packed pop.',
                         price: 600,
                         brand: 'Burton',
                         image: File.open('app/assets/images/burton_process.jpg')
                       },
-                      { name: 'NEIGHBORHOOD Throwback',
+                      { name: 'Throwback',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.